src/save_options/__init__.py

```diff
@@ -0,0 +1,71 @@
+import save_options.hooks  # noqa: F401  # pyright: ignore[reportUnusedImport]
+from mods_base.mod_list import base_mod
+from save_options.options import (
+    BoolSaveOption,
+    HiddenSaveOption,
+    SaveOption,
+    SliderSaveOption,
+    SpinnerSaveOption,
+)
+from save_options.registration import register_save_options
+
+__all__: tuple[str, ...] = (
+    "BoolSaveOption",
+    "HiddenSaveOption",
+    "SaveOption",
+    "SliderSaveOption",
+    "SpinnerSaveOption",
+    "__author__",
+    "__version__",
+    "__version_info__",
+    "register_save_options",
+)
+
+__version_info__: tuple[int, int] = (1, 0)
+__version__: str = f"{__version_info__[0]}.{__version_info__[1]}"
+__author__: str = "bl-sdk"
+
+"""This library allows for any arbitrary JSON encodable data to be saved in the character's .sav
```

Comment:
Just go with "in the character's save file" to avoid the spelling warning.

nitpick: if it's not a one line docstring, the triple quotes go on their own lines